### PR TITLE
Display scopeless user attributes with the improved search field

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject

### Related Issues
- None

### Related PRs
- None

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx [hunk 1]
```diff
@@ -318,13 +318,14 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
     const getExternalClaimsGroupedByScopes = () => {
         const tempClaims = [ ...externalClaims ];
         const updatedScopes = [];
-
+        const scopedClaims = [];
```
Comment (savindi7) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx:321]: Shall we type this?

Comment (savindi7) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx:321]:
```suggestion
        const scopedClaims: ExtendedExternalClaimInterface[] = [];
```

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx [hunk 2]
```diff
@@ -347,10 +348,35 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
...
                     };
-    
+     
                     updatedScopes.push(updatedScope);
                 }
             });
+ 
+            const scopelessClaims = tempClaims.filter((tempClaim) => !scopedClaims.includes(tempClaim));
+            const updatedScopelessClaims = [];
+            let isScopelessClaimRequested = false;
```
Comment (savindi7) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx:358]:
```suggestion
            let isScopelessClaimRequested: boolean = false;
```

**Review by savindi7 — DISMISSED** [review]

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx [hunk 3]
```diff
@@ -501,13 +518,30 @@ export const AttributeSelectionOIDC: FunctionComponent<AttributeSelectionOIDCPro
...
+                    ) : (
+                        <Header.Content>
+                            { scope.displayName }
+                            <Header.Subheader>
+                                <Code withBackground>{ scope.name }</Code>
+                                { " " + (scope.description ?? "") }
+                            </Header.Subheader>
+                        </Header.Content>
+                    )
+                }
```
Comment (thivi) [inline apps/console/src/features/applications/components/settings/attribute-management/attribute-selection-oidc.tsx:544]: Shall we format this properly?

**Review by thivi — DISMISSED** [review]

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx (hunk 1 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3537/commits/fe3791422914bf046739797bffccb53b7a9e248d

File: apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx (hunk 2 above)
Comment (mpmadhavig) [reply apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx]: resolved with https://github.com/wso2/identity-apps/pull/3537/commits/fe3791422914bf046739797bffccb53b7a9e248d

**Comment by mpmadhavig** [comment]
A new PR with all the changes related to the issue will be added later.

**Pull request closed without merging**